libsolutil/CommonIO.h

```diff
@@ -48,6 +48,46 @@ inline std::ostream& operator<<(std::ostream& os, bytes const& _bytes)
 namespace util
 {
 
+namespace detail
+{
+
+template <typename Predicate>
+struct RecursiveFileCollector
+{
+	Predicate predicate;
+	std::vector<boost::filesystem::path> result {};
```

Comment:
It looks like an empty code block if you write it this way :)
```suggestion
	std::vector<boost::filesystem::path> result{};
```